commands/quickref/quickref.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+const path = require('path');
+
+const getFileList = (dirName) => {
+  const fileList = fs.readdirSync(dirName);
+  return fileList || [];
+};
+
+const loadReferences = (filename) => {
+  const content = fs.readFileSync(filename, 'utf8');
+  return content || '';
+};
+
+const references = {};
+
+module.exports = {
+  usage: [
+    'quickref <topic> - displays quick reference for <topic>',
+    'help - list known references',
```

Comment:
From the implementation below I'm guessing you meant `quickref - list known references` (i.e. `!bot quickref` will show the list of topics)?